package-lock.json

```diff
@@ -779,7 +781,7 @@
       "version": "0.3.9",
       "resolved": "https://registry.npmjs.org/@jridgewell/trace-mapping/-/trace-mapping-0.3.9.tgz",
       "integrity": "sha512-3Belt6tdc8bPgAtbcmdtNJlirVoTmEb5e2gC94PnkwEW9jI6CAHUeoG85tjWP5WquqfavoMtMwiG4P926ZKKuQ==",
-      "dev": true,
+      "devOptional": true,
```

Comment:
these changes shouldn't be a part of the changeset

Reply:
ohhh I was installing with --force, and not --legacy-peer-deps. Will reinstall and push!